security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -88,7 +88,7 @@ type sdsConnection struct {
 	pushChannel chan *sdsEvent
 
 	// SDS streams implement this interface.
-	stream v2.DiscoveryStream
+	stream xds.DiscoveryStream
```

Comment:
Here the PR changes "v2.DiscoveryStream" to "xds.DiscoveryStream". DiscoveryStream is specific to "istio.io/istio/pilot/pkg/proxy/envoy/v2" but "xds.DiscoveryStream" does not contain the version. Can you explain how the version is represented in "xds.DiscoveryStream"?

Reply:
`DiscoveryStream` is an interface that is not version specific and but currently is under v2. Now I moved that code to xds package in this PR